src/Symfony/Component/Messenger/Stamp/RedeliveryStamp.php

```diff
@@ -22,14 +22,30 @@ final class RedeliveryStamp implements StampInterface
     private $retryCount;
     private $redeliveredAt;
     private $exceptionMessage;
-    private $flattenException;
+    private $flattenedException;
```

Comment:
renaming the property probably breaks the deserialization of existing stamps in a message

Reply:
You're right. This was a mistake on my part when I removed and readded the attribute.